doc/release/v3_1_0.md

```diff
@@ -17,9 +17,47 @@ New Features
+  src/libYARP_dev/include/yarp/dev/ImplementPidControl.h (from src/libYARP_dev/include/yarp/dev/IPidControlImpl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementPositionControl.h (from src/libYARP_dev/include/yarp/dev/IPositionControlImpl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementPositionControl2.h (from src/libYARP_dev/include/yarp/dev/IPositionControl2Impl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementPositionDirect.h (from src/libYARP_dev/include/yarp/dev/IPositionDirectImpl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementVelocityControl.h (from src/libYARP_dev/include/yarp/dev/IVelocityControlImpl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementVelocityControl2.h (from src/libYARP_dev/include/yarp/dev/IVelocityControl2Impl.h)
+  src/libYARP_dev/include/yarp/dev/ImplementVirtualAnalogSensor.h (from src/libYARP_dev/include/yarp/dev/IVirtualAnalogSensorImpl.h)
+  src/libYARP_dev/src/ImplementAmplifierControl.cpp (from src/libYARP_dev/src/IAmplifierControlImpl.cpp)
+  src/libYARP_dev/src/ImplementAxisInfo.cpp (from src/libYARP_dev/src/IAxisInfoImpl.cpp)
+  src/libYARP_dev/src/ImplementControlCalibration.cpp (from src/libYARP_dev/src/IControlCalibrationImpl.cpp)
+  src/libYARP_dev/src/ImplementControlLimits.cpp (from src/libYARP_dev/src/IControlLimitsImpl.cpp)
+  src/libYARP_dev/src/ImplementControlMode.cpp (from src/libYARP_dev/src/ControlModeImpl.cpp)
+  src/libYARP_dev/src/ImplementCurrentControl.cpp (from src/libYARP_dev/src/CurrentControlImpl.cpp)
+  src/libYARP_dev/src/ImplementEncoders.cpp (from src/libYARP_dev/src/IEncodersImpl.cpp)
+  src/libYARP_dev/src/ImplementEncodersTimed.cpp (from src/libYARP_dev/src/IEncodersTimedImpl.cpp)
+  src/libYARP_dev/src/ImplementImpedanceControl.cpp (from src/libYARP_dev/src/ImpedanceControlImpl.cpp)
+  src/libYARP_dev/src/ImplementInteractionMode.cpp (from src/libYARP_dev/src/IInteractionModeImpl.cpp)
+  src/libYARP_dev/src/ImplementMotor.cpp (from src/libYARP_dev/src/IMotorImpl.cpp)
+  src/libYARP_dev/src/ImplementMotorEncoders.cpp (from src/libYARP_dev/src/IMotorEncodersImpl.cpp)
+  src/libYARP_dev/src/ImplementPWMControl.cpp (from src/libYARP_dev/src/PWMControlImpl.cpp)
+  src/libYARP_dev/src/ImplementPidControl.cpp (from src/libYARP_dev/src/IPidControlImpl.cpp)
+  src/libYARP_dev/src/ImplementPositionControl.cpp (from src/libYARP_dev/src/IPositionControlImpl.cpp)
+  src/libYARP_dev/src/ImplementPositionDirect.cpp (from src/libYARP_dev/src/IPositionDirectImpl.cpp)
+  src/libYARP_dev/src/ImplementRemoteCalibrator.cpp (from src/libYARP_dev/src/RemoteCalibratorImpl.cpp)
+  src/libYARP_dev/src/ImplementRemoteVariables.cpp (from src/libYARP_dev/src/IRemoteVariablesImpl.cpp)
+  src/libYARP_dev/src/ImplementTorqueControl.cpp (from src/libYARP_dev/src/TorqueControlImpl.cpp)
+  src/libYARP_dev/src/ImplementVelocityControl.cpp (from src/libYARP_dev/src/IVelocityControlImpl.cpp)
+  src/libYARP_dev/src/ImplementVirtualAnalogSensor.cpp (from src/libYARP_dev/src/VirtualAnalogSensorImpl.cpp)
```

Comment:
@randaz81 For next time, please note that:
* the release notes are for the final user, there is no need to list all the `.cpp` files renamed, only the changes to the `.h` files are important, but the *internal* path is not... (i.e. `yarp/dev/ImplementControlLimits.h (from yarp/dev/IControlLimitsImpl.h)` is enough).
* Use 80 characters word wrap.
* Wrap file names, methods, etc. using the "``" character, so that they are rendered using the `monospaced font`
* The list of files in the `CMakeLists.txt` are sorted alphabetically, please keep it that way.